lib/action_policy/graphql/behaviour.rb

```diff
@@ -36,7 +36,7 @@ def self.included(base)
 
         base.authorize :user, through: :current_user
 
-        if base.respond_to?(:field_class)
+        if base.respond_to?(:field_class) && !(base.field_class < ActionPolicy::GraphQL::AuthorizedField)
```

Comment:
Let's cover all the cases

```suggestion
        if base.respond_to?(:field_class) && !(base.field_class <= ActionPolicy::GraphQL::AuthorizedField)
```

Reply:
I do believe `ActionPolicy::GraphQL::AuthorizedField` could not be used as field class

Reply:
since it's a module